### DISCUSSION TITLE
This library download MongoDB shell? and use it ?

### DISCUSSION THREAD

**Discussion by Mehrdad-Miri130** [typegoose/mongodb-memory-server#485] [Q&A] [answered]
## Versions

- NodeJS: 14.0.0
- mongodb-memory-server-*: 6.9.6
- mongodb: 0.0.0
- mongoose: 5.12.13 
- system: Windows 10

package: mongo-memory-server 

## how can I run the mongoexpert command with this library to export data as JSON files?
I want to add a collection to Db and export it as a JSON file, but can't run the command without MongoDB shell
mongoexport -h HOST:PORT -d DB_NAME -c COLLECTION_NAME -o PATH
can I run the above command with this library?

**Comment by hasezoey** [comment, marked as answer]
yes this package downloads the archive, but only extracts & keeps the `mongod` binary

(and no, the others will not be added to the extraction)

**Reply by Mehrdad-Miri130** [reply]
tnx
